mmv1/products/gkehub2/Feature.yaml

```diff
@@ -0,0 +1,161 @@
+# Copyright 2023 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the License);
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Resource
+name: 'Feature'
+base_url: projects/{{project}}/locations/{{location}}/features
+create_url: projects/{{project}}/locations/{{location}}/features?featureId={{name}}
+update_url: projects/{{project}}/locations/{{location}}/features/{{name}}
+self_link: projects/{{project}}/locations/{{location}}/features/{{name}}
+update_verb: :PATCH
+update_mask: true
+description: |
+  Feature represents the settings and status of any Hub Feature.
+references: !ruby/object:Api::Resource::ReferenceLinks
+  guides:
+    'Registering a Cluster': 'https://cloud.google.com/anthos/multicluster-management/connect/registering-a-cluster#register_cluster'
+  api: 'https://cloud.google.com/anthos/fleet-management/docs/reference/rest/v1/projects.locations.features'
+async: !ruby/object:Api::OpAsync
+  operation: !ruby/object:Api::OpAsync::Operation
+    kind: 'gkehub2#operation'
+    path: 'name'
+    base_url: '{{op_id}}'
+    wait_ms: 1000
+  result: !ruby/object:Api::OpAsync::Result
+    path: 'response'
+    resource_inside_response: true
+  status: !ruby/object:Api::OpAsync::Status
+    path: 'done'
+    complete: 'true'
+    allowed:
+      - 'true'
+      - 'false'
+  error: !ruby/object:Api::OpAsync::Error
+    path: 'error/errors'
+    message: 'message'
+iam_policy: !ruby/object:Api::Resource::IamPolicy
```

Comment:
I honestly could not think a better way to solve the iam tests problem then handwrite the iam tests. (I reused the generated one and modified a bit: https://gist.github.com/shuyama1/cf39cf807e091a64b59231cd96f3f0e8, you can copy it to your handwritten iam test file if needed)

After that, we can add `skip_test: true` to `gkehub_feature_basic` block, as we will rely testing on handwritten tests